src/megameklab/com/printing/PrintProtomech.java

```diff
@@ -0,0 +1,188 @@
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ */
+package megameklab.com.printing;
+
+import megamek.common.*;
+import megameklab.com.MegaMekLab;
+import org.apache.batik.anim.dom.SVGLocatableSupport;
+import org.apache.batik.util.SVGConstants;
+import org.w3c.dom.Element;
+
+import java.util.StringJoiner;
+
+/**
+ * Lays out a record sheet block for a single protomech
+ */
+public class PrintProtomech extends PrintEntity {
+
+    private final Protomech proto;
+    private final int unitIndex;
+
+    /**
+     * Creates an SVG object for the record sheet
+     *
+     * @param proto        The protomech to print
+     * @param startPage    The print job page number for this sheet
+     * @param unitIndex    The index of this unit on the page
+     * @param options      Overrides the global options for which elements are printed
+     */
+    public PrintProtomech(Protomech proto, int startPage, int unitIndex, RecordSheetOptions options) {
+        super(startPage, options);
+        this.proto = proto;
+        this.unitIndex = unitIndex;
+    }
+
+    /**
+     * Creates an SVG object for the record sheet using the global printing options
+     *
+     * @param proto        The protomech to print
+     * @param startPage    The print job page number for this sheet
+     * @param unitIndex    The index of this unit on the page
+     */
+    public PrintProtomech(Protomech proto, int startPage, int unitIndex) {
+        this(proto, startPage, unitIndex, new RecordSheetOptions());
+    }
+
+    @Override
+    protected String getSVGFileName(int pageNumber) {
+        if (proto.isQuad()) {
+            return "protomech_quad.svg";
+        } else if (proto.isGlider()) {
+            return "protomech_glider.svg";
+        } else {
+            return "protomech_biped.svg";
+        }
+    }
+
+    @Override
+    protected Entity getEntity() {
+        return proto;
+    }
+
+    @Override
+    protected boolean isCenterlineLocation(int loc) {
+        return false;
+    }
+
+    @Override
+    protected String getRecordSheetTitle() {
+        // Not used
+        return "";
+    }
+
+    @Override
+    protected void writeTextFields() {
+        super.writeTextFields();
+        setTextField(PROTOMECH_INDEX, "PROTOMECH " + (unitIndex + 1));
+        splitName();
+        if (getEntity().hasUMU()) {
+            setTextField(LBL_JUMP, "Underwater:");
+            setTextField(MP_JUMP, getEntity().getAllUMUCount());
+        }
+        printTorsoCritChart();
+        if (!proto.hasMainGun()) {
+            hideElement(MAIN_GUN_ARMOR);
+            hideElement(MAIN_GUN_SHADOW);
+            hideElement(MAIN_GUN_TEXT);
+        }
+        if (proto.hasUMU()) {
+            setTextField(LBL_JUMP, "Underwater");
```

Comment:
This seems redundant and/or replaces what's on line 90.

Reply:
I must have written them on different days.